python/cuml/feature_extraction/_tfidf_vectorizer.py

```diff
@@ -256,3 +256,13 @@ def transform(self, raw_documents):
         """
         X = super().transform(raw_documents)
         return self._tfidf.transform(X, copy=False)
+
+    def get_feature_names(self):
```

Comment:
Just after approving remembered that I forgot to ask, could you add a very simple pytest so that this method is covered?

Reply:
Sure